src/controllers/expense.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  getExpense,
+  deleteExpense,
+  createNewExpense,
+  updateExpense,
+} = require('../services/expenses');
+
+const getAllExpensesController = (req, res) => {
+  const { userId, category, from, to } = req.query;
+
+  if (!category || !userId || !from || !to) {
+    res.sendStatus(422);
+
+    return;
```

Comment:
Why do we need to return 422 error here? It's a method for getting all todos, and it should work even if no filter params is passed